pkg/storage/sds/restclient/restclient.go

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package restclient
```

Comment:
please rename to just `client`

Reply:
The reason I named it "restclient" instead of just "client" is because my next step is to add a "full" SDS client that incorporates the crypto ability and the REST stuff, and I was going to make that "full" client type contain a SDS REST client. This allows me to make the files smaller and also allows others to use just the REST client separately from the crypto stuff. Then they can construct Encrypted Documents using their own crypto code. Does that approach make sense?

Reply:
To clarify - I was planning on naming the package with the full SDS client just "client", unless you have a better name.

Or maybe you're thinking of having it all in one package, but different file names? Hmm...

Reply:
I'm not entirely clear what you mean by "full SDS client that incorporates the crypto ability":

* we need a `Store` implementation that stores stuff in the SDS
* encryption is required regardless of the `Store` implementation
* therefore I'm not sure what exact "crypto abilities" this "full SDS client" will have

Let's discuss this to see if I understand the design correctly.

Reply:
(marking as resolved since this conversation was based on a much older code design. It's not really applicable in the same way anymore)